Gemfile

```diff
@@ -1,89 +1,41 @@
 source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '2.6.1'
+ruby '2.6.4'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '~> 5.1.3'
+gem 'rails', '~> 6.0.0'
 # Use postgresql as the database for Active Record
-gem 'pg', '~> 1.1'
+gem 'pg', '>= 0.18', '< 2.0'
 # Use Puma as the app server
-gem 'puma', '~> 3.12'
+gem 'puma', '~> 3.11'
+# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
+# gem 'jbuilder', '~> 2.7'
+# Use Redis adapter to run Action Cable in production
+# gem 'redis', '~> 4.0'
+# Use Active Model has_secure_password
+# gem 'bcrypt', '~> 3.1.7'
 
-gem 'therubyracer', platforms: :ruby
+# Use Active Storage variant
+# gem 'image_processing', '~> 1.2'
 
-# Sidekiq
```

Comment:
We could add all the missing dependencies in another PR

Reply:
I think it would be easier to add everything that we need in this PR so it's clear to see the differences that were made because of this migration. But maybe it won't be as cleaner as doing it in another PR :thinking:

Reply:
I strongly suggest we add the required dependencies in another PR

Reply:
Ok, I'd create a new PR adding all the gems that were removed but are needed for this bootstrap. :)